common/arch/sdl/key.cpp

```diff
@@ -587,6 +587,11 @@ void key_flush()
 		restore_sticky_key(keystate, key);
 }
 
+void event_keycommand_send(int key) {
+	const d_event_keycommand event{EVENT_KEY_COMMAND, unsigned(key)};
```

Comment:
Instead of using a function-style cast, make the parameter the right type on entry.